README.md

```diff
@@ -43,7 +43,30 @@ it, you will have to run `cmake` manually like so:
   ```bash
   cmake -DCMAKE_INSTALL_PREFIX=/path/of/install/dir
  ```
- 
+
+The target of the complex_bessel library is exported
+as `complex_bessel::complex_bessel` to a package configuration file for this
+library.
+
+After installation you can find this library in your
+project CMakeLists.txt with:
+  ```cmake
+  find_package(complex_bessel)
+```
+or you can just put a copy of `complex_bessel` source code with zero changes
```

Comment:
Remove "zero changes". They can just put a copy of the source.

Reply:
done